src/Components/Components/src/ParameterView.cs

```diff
@@ -212,7 +212,7 @@ internal void CaptureSnapshot(ArrayBuilder<RenderTreeFrame> builder)
         /// </summary>
         /// <param name="parameters">The <see cref="IDictionary{TKey, TValue}"/> with the parameters.</param>
         /// <returns>A <see cref="ParameterView"/>.</returns>
-        public static ParameterView FromDictionary(IDictionary<string, object> parameters)
+        public static ParameterView FromDictionary(IDictionary<string, object?> parameters)
```

Comment:
This is a breaking change. I'll send an API review email to discuss this the change with the team.